localstack/services/apigateway/invocations.py

```diff
@@ -215,14 +217,28 @@ def apply_response_parameters(invocation_context: ApiInvocationContext):
     int_responses = integration.get("integrationResponses") or {}
     if not int_responses:
         return response
+
+    integration_type_orig = integration.get("type") or integration.get("integrationType") or ""
```

Comment:
Interesting, because `_PROXY` integrations can't have integration responses, it should have returned on [line](https://github.com/localstack/localstack/pull/8509/files#diff-ae146813864a8ec6b0ac068c7ea1098520a01dd30eeb6063458e99c385428148R218)

Reply:
You are right. However, it might be possible to still do this on LS, it is just an additional guard here.